packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -76,6 +77,16 @@ export const mapUserToApplicationContextUser = (user?: TFetchedUser) => {
   };
 };
 
+// Adjust certain fields which depend e.g. on the origin
+export const mapEnvironmentToApplicationContextEnvironment = <
+  TApplicationEnvironment extends {}
+>(
+  environment: TApplicationEnvironment
+) => ({
+  mcApiUrl: getMcApiUrl(),
```

Comment:
Shouldn't this be after `...environment`?

Also, it might be more clear to pass the `environment` object to the function `getMcApiUrl(environment)` instead of using the `window.app` value.

Reply:
Yeah. Can adjust the fn in this case. Too late now. Will do tmr.